### PR TITLE
feat: lwseverity package

### PR DESCRIPTION
**Pull request by hazedav**
## Summary
As a Lacework developer I would like a package to make working with Lacework severities easy as pie (or tacos).

- Normalizing
- Sorting
- Filtering

## How did you test this change?
Unit testing

## Issue
https://lacework.atlassian.net/browse/ALLY-1263

### CODE REVIEW

File: lwseverity/severity.go [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+// Take a string representation of Lacework severity and
+// return it's normalized integer and string values
+// Critical Severity      => 1, "Critical"
+// High Severity          => 2, "High"
+// Medium Severity        => 3, "Medium"
+// Low Severity           => 4, "Low"
+// Informational Severity => 5, "Info"
+// Unknown Severity       => 0, "Unknown"
+func SeverityToProperTypes(severity string) (int, string) {
```
Comment (hazedav) [inline lwseverity/severity.go:39]: Now that we have this abstracted is `Normalize` a better/simpler name for this...

Thoughts @afiune

Comment (hazedav) [reply lwseverity/severity.go]: I went ahead with this.

**Review by dmurray-lacework — APPROVED** [review]
Nice!

File: lwseverity/severity.go (hunk 1 above)
Comment (afiune) [reply lwseverity/severity.go]: Great stuff!

**Pull request merged**